### PR TITLE
Add support for login items to 1password extension

### PR DESCRIPTION
**Pull request by henrik-dmg**
## Description

This PR adds support for using the item type `PASSWORD` with the user-configured actions. It supports the same actions as `LOGIN`, except for `copy-username`

## How to Test

- add a `Login` item in 1Password
- configure `Copy username` as the primary action
- activate the extension and search for your login
- `Copy username` should be shown as primary action and copy the login to the clipboard when pressing enter

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/1password/.gitignore
Comment (henrik-dmg) [inline extensions/1password/.gitignore:1]: No idea how this ended up in the diff 🤔

Comment (pernielsentikaer) [reply extensions/1password/.gitignore:1]: We really want it to be there 🙂

**Comment by pernielsentikaer** [comment]
Seems to be working as expected for me @henrik-dmg - could re-add .gitignore and a CHANGELOG entry 🙂

@khasbilegt do you mind checking this?

**Comment by henrik-dmg** [comment]
Could this be merged if nobody has further comments? 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**